### PR TITLE
[rabbitmq] Set securityContext with correct user

### PR DESCRIPTION
**Pull request by desaintmartin**
tl;dr Latest version of rabbitmq helm chart crashes because of permission issues.

Allows kubernetes to chown a potential PersistentVolume to this user.
The most recent version of rabbitmq bitnami docker images uses the user 1001: https://github.com/bitnami/bitnami-docker-rabbitmq/blob/3.7.7-debian-9-r21/3.7/debian-9/Dockerfile\#L33

Previously, with 2.0.1 and a new persistentvolume, the boot script was not able to chown echo `/opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie` because of permission issue (PersistentVolume is owned by root by default in Kubernetes/Docker).

ping @bitnami-bot (is there anybody behind you, dear bot?)

### CODE REVIEW

**Comment by desaintmartin** [comment]
ping @prydonius @tompizmor @sameersbn @juan131

**Comment by desaintmartin** [comment]
Am I the only one to have this blocking issue?

**Comment by javsalgar** [comment]
Hi,

Could you let us know a bit more about the issue? Does it happen when simply launching helm install stable/rabbitmq? Do you have the latest version of the chart (i.e. 'helm repo update`)? Which is your environment?

**Comment by javsalgar** [comment]
In any case, LGTM

**Comment by desaintmartin** [comment]
Hello,
Using the latest version of the chart, without my changes, using a standard values.yaml and enabling persistence, the deployed Pod reads:
`bash: line 1: /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie: Permission denied`

This is because Kubernetes (through Docker?) creates new PersistentVolumes with root as owner. We thus need to change it since we do no longer run as root.

Let me do the requested changes.

**Comment by desaintmartin** [comment]
Done!

**Comment by carrodher** [comment]
Hi @desaintmartin, some conflicts appear and I can't merge the PR, can you take a look? After that I should be able to merge your changes

**Comment by patrickleet** [comment]
looks like same changes were made in this PR https://github.com/helm/charts/pull/6946

**Comment by nicklv** [comment]
sudo chown -R  1001 xxxxx
sudo chown R  1001 xxxxxx

**Comment by hudvin** [comment]
Still reproducable.
